omni/omni.py

```
@@ -0,0 +1,379 @@
+        return flagged, category_scores
+
+    async def handle_moderation(self, message, category_scores, image_flagged):
+        guild = message.guild
+        timeout_duration = await self.config.guild(guild).timeout_duration()
+        log_channel_id = await self.config.guild(guild).log_channel()
+
+        # Delete the message
+        try:
+            await message.delete()
+        except discord.NotFound:
+            pass  # Handle cases where the message is already deleted
```

Ruff (0.8.2)

Use `contextlib.suppress(discord.NotFound)` instead of `try`-`except`-`pass`

(SIM105)